playground/extensions/extensions-1-tools.md

```diff
@@ -20,12 +20,12 @@ Example of a project using the Graphic Entity Module and the End Screen Module:
 <dependency>
 	<groupId>com.codingame.gameengine</groupId>
 	<artifactId>module-entities</artifactId>
-	<version>3.4.1</version>
+	<version>4.3.0</version>
```

Comment:
Should we use `${gamengine.version}` here instead ?

Reply:
yes that would be preferable